### PR TITLE
Gamedrake 

### PR DESCRIPTION
**Pull request by slimmoe**

### CODE REVIEW

**Comment by frozaum** [comment]
why you did a different pr?

File: gamedrake/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "author": {
+      "name": "Slim Moe",
+      "id": "686244278304899077"
+    },
+    "service": "Gamedrake",
+    "description": {
+        "en": "Game Drake is the platform for players to learn about games.",
+        "tr": "Gamedrake, oyunlar hakkında bilgi edinebileceğiniz oyuncu platformu." 
+       },
+    "url": ["gamedrake.com"],
```
Comment (Cairo2k18) [inline gamedrake/dist/metadata.json:11]: Please make this a string, it does not need to be an array as it only contains one thing.

`["gamedrake.com"]` -> `"gamedrake.com"`

Comment (Cairo2k18) [inline gamedrake/dist/metadata.json:11]: Please make this a string, it does not need to be an array as it only contains one thing.

`["gamedrake.com"]` -> `"gamedrake.com"`

File: gamedrake/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "version": "1.1.1",
```
Comment (Cairo2k18) [inline gamedrake/dist/metadata.json:12]: This should be version `1.0.0` as it is a new presence.

File: gamedrake/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "Slim Moe",
+      "id": "686244278304899077"
+    },
+    "service": "Gamedrake",
+    "description": {
```
Comment (Bas950) [inline gamedrake/dist/metadata.json:7]: Service name and folder name must be the same, caps letters, spaces, etc. included.

File: gamedrake/presence.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+var presence = new Presence({
+  clientId: "707525132322865152"
+});
+var browsingStamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+  const data = {
```
Comment (Cairo2k18) [inline gamedrake/presence.ts:6]: add presenceData type for this

**Comment by doomlerd** [comment]
Will be closed in 3 days.

**Pull request closed without merging**